docs/releases/4.0.md

```diff
@@ -17,8 +17,7 @@ depth: 1
  * Add `full_url` to the API output of `ImageRenditionField` (Paarth Agarwal)
  * Use `InlinePanel`'s label when available for field comparison label (Sandil Ranasinghe)
  * Drop support for Safari 13 by removing left/right positioning in favour of CSS logical properties (Thibaud Colas)
- * Use `FormData` instead of jQuery's `form.serialize` when editing documents or images just added so that additional fields can be better supported (Stefan Hammer)
```

Comment:
You've clobbered some existing release notes here :-)

Reply:
Thanks! Must've messed it up during a rebase :D